POI/places.go

```diff
@@ -46,7 +46,7 @@ type Place struct {
 	Address          Address        `bson:"address"`
 	FormattedAddress string         `bson:"formatted_address"`
 	Location         Location       `bson:"location"`
-	PriceLevel       int            `bson:"price_level"`
+	PriceLevel       PriceLevel     `bson:"price_level,string"`
```

Comment:
Curious what does the newly added ",string" mean here? I know that "price_level" is BSON/JSON tag

Reply:
Good catch! Initially I wanted to make the level a string, removed it.